packages/app/src/templates/components/templates/templates.interface.ts

```diff
@@ -5,11 +5,14 @@
 
 import { PayloadAction } from "@reduxjs/toolkit"
 import { Template, NewTemplate } from "App/templates/dto"
+import { TemplateDeletingState } from "App/templates/constants"
 
 export interface TemplatesProps {
   templates: Template[]
   loading: boolean
   error: Error | string | null
+  deletingState: TemplateDeletingState | null
   createTemplate: (arg: NewTemplate) => Promise<PayloadAction<any>>
-  deleteTemplates: (ids: string[]) => void
+  deleteTemplates: (arg: string[]) => Promise<PayloadAction<any>>
```

Comment:
Maybe `unknown` instead of `any` would be a safer option? Not forcing though, seems to be not super important in that case.